public/version-aliases/go.json

```diff
@@ -18,6 +18,8 @@
   "1.8.x": "1.8.7",
   "1.9.4": "1.9.4",
   "1.9.x": "1.9.4",
+  "1.10": "1.10",
+  "1.10.x": "1.10",
   "1.2": "1.2.2",
   "go1": "go1"
-}
+}
```

Comment:
It's a tiny thing, but please restore the newline here.

Reply:
I fixed it by https://github.com/travis-ci/travis-build/pull/1310/commits/2bcf8f7fc4e785643485fa535e35367e500d1bff .

`bundle exec rake public/version-aliases/go.json` has removed the newline here.